npm i after installing package fails

**Describe the bug**
Error show when running `npm install` after installing the package in a repo.

**To Reproduce**
Steps to reproduce the behavior:
1. Install de package `npm i @asigloo/vue-dynamic-forms`
2. Run `npm install` again

**Expected behavior**
No Errors

**Log**

```
npm ERR! path /Users/alvarosaburido/as/test/node_modules/@asigloo/vue-dynamic-forms
npm ERR! code EISGIT
npm ERR! git /Users/alvarosaburido/as/test/node_modules/@asigloo/vue-dynamic-forms: Appears to be a git repo or submodule.
npm ERR! git     /Users/alvarosaburido/as/test/node_modules/@asigloo/vue-dynamic-forms
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/alvarosaburido/.npm/_logs/2020-05-04T14_47_46_677Z-debug.log
```
**Posible fixes** 
https://github.com/APSL/react-native-version-number/issues/13#issuecomment-341059894

--- a/.npmignore
+++ b/.npmignore
@@ -7,6 +7,8 @@ babel.config.js
 vue.config.js
 postcss.config.js
 .prettierrc
+.git
+**/.git/
 yarn.lock
 /dev
 /docs

